docs/configuration.md

```diff
@@ -1398,6 +1398,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.listenerbus.eventqueue.size</code></td>
+  <td>10000</td>
+  <td>
+    Capacity for event queue in Spark listener bus, must be greater than 0. Consider increasing
+    value (e.g. 20000) if listener events are dropped, because of the rate at which tasks are being
```

Comment:
Will do, thanks. I am going to update description to this:
```
Capacity for event queue in Spark listener bus, must be greater than 0. 
Consider increasing value (e.g. 20000) if listener events are dropped. 
Increasing this value will result in the driver using more memory.
```

Is it okay, if I make this change?